conans/client/command.py

```diff
@@ -583,10 +584,11 @@ def config(self, *args):
                 self._outputer.json_output({"home": conan_home}, args.json, os.getcwd())
             return conan_home
         elif args.subcommand == "install":
-            verify_ssl = get_bool_from_text(args.verify_ssl)
-            return self._conan.config_install(args.item, verify_ssl, args.type, args.args,
-                                              source_folder=args.source_folder,
-                                              target_folder=args.target_folder)
+            if is_config_install_scheduled(self._conan):
```

Comment:
This feature is not for the ``conan config install`` command. That command should execute whenever it is called. No questions, no sched, it just runs.

This feature is for every other command. To automatically call "conan config install" even when you do a "conan search", if it is scheduled.

Reply:
I'm curious why don't we use on API level, as all hook call are executed directly on same level as the API.

Reply:
To not be recursive on the ``conan_config_install`` apil call.
This is not an "api" thing, the "api" is already there. It is an "app" thing.